### PR TITLE
fix word

### PR DESCRIPTION
**Pull request by tianheg**
### Prerequisites

Put an `x` into the box(es) that apply:

- [x] This pull request fixes a bug.
- [ ] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

fix a word: `colorsheme` to `colorscheme`

### Issues Resolved

None

### Checklist

Put an `x` into the box(es) that apply:

#### General

- [x] Describe what changes are being made
- [ ] Explain why and how the changes were necessary and implemented respectively
- [ ] Reference issue with `#<ISSUE_NO>` if applicable

#### Resources

- [ ] If you have changed any SCSS code, run `make release` to regenerate all CSS files

#### Contributors

- [x] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

File: assets/js/dark-mode.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const body = document.body;
 const darkModeToggle = document.getElementById('dark-mode-toggle');
 const darkModeMediaQuery = window.matchMedia('(prefers-color-scheme: dark)');
 
-// Check if user preference is set, if not check value of body class for light or dark else it means that colorsheme = auto
+// Check if user preference is set, if not check value of body class for light or dark else it means that colorscheme = auto
```
Comment (luizdepra) [inline assets/js/dark-mode.js:5]: Nice catch! xD

Comment (tianheg) [reply assets/js/dark-mode.js:5]: happy to hear that

**Comment by luizdepra** [comment]
There is a conflict at `CONTRIBUTORS.md`. Could you please fix it?

**Comment by tianheg** [comment]
@luizdepra OK

**Comment by luizdepra** [comment]
Thank you.

**Pull request merged**